engine/include/cubos/engine/font/font.hpp

```diff
@@ -0,0 +1,38 @@
+/// @file
+/// @brief Struct @ref cubos::engine::Font.
+/// @ingroup font-plugin
+
+#pragma once
+
+#include <vector>
+
+#include <cubos/core/memory/stream.hpp>
+#include <cubos/core/reflection/reflect.hpp>
+
+#include <cubos/engine/api.hpp>
+
+namespace cubos::engine
+{
+    /// @brief Asset containing font data.
```

Comment:
I would explain a little bit better what this asset represents, so that people can understand the difference from the `FontAtlas` asset.